x-pack/legacy/plugins/rollup/public/services/api.js

```diff
@@ -9,6 +9,8 @@ export const setHttpClient = client => {
 };
 
 export async function getRollupIndices() {
-  const response = await httpClient.get(`api/rollup/indices`);
-  return response.data || {};
+  // '..' exists to ignore randomized app basename.
+  // TODO: Find a way to remove '..'
+  const response = await httpClient.get(`../api/rollup/indices`);
+  return response || {};
```

Comment:
It's written in this way because without '..', kibana tries to get data from 'localhost:5620/app/api/rollup/indices'. We need to remove **app** from here. 

But I couldn't find a way to ignore randomized kibana basename. This is really hacky. 

I hope there is a good alternative for this.